source/crud/crud.rst

```diff
@@ -1612,6 +1615,7 @@ Find And Modify
      * This option is sent only if the caller explicitly provides a value. The default is to not send a value.
      * This option is only supported by servers >= 4.4. Older servers >= 4.2 will report an error for using this option.
      * For servers < 4.2, the driver MUST raise an error if the caller explicitly provides a value.
+     * For unacknowledged writes using OP_QUERY or OP_MSG for servers < 4.4, the driver MUST raise an error if the caller explicitly provides a value.
```

Comment:
The Q&A that Jeremy added: "Why are client-side errors raised for some unsupported options?" says:
> The spec does not explicitly refer to unacknowledged writes using OP_QUERY primarily because a response document is always returned and drivers generally would not consider using OP_QUERY precisely for that reason.

So I think this can just refer to OP_MSG. Same for the other findAndModify based APIs below.

Reply:
Great, thank you.